### ISSUE TITLE
[advice] table schema change

### ISSUE THREAD

**Issue by AnemoneIndicum** [StarRocks/starrocks#13110]
**There are some shortcomings in the table structure change of StarRocks. When using it, I feel it is not elegant enough.**

- The table schema change with a big table, it's maybe a timeout.

- That StarRocks database performs schema changes which are not a simultaneous way, which to create an asynchronous task executed by schedule. So,In 'Flink CDC' synchronization task, we must obtain a result on whether the table struct change has been completed despite it is costly,  otherwise, The new data collected is lost .

- Using the command 'SHOW ALTER TABLE COLUMN from xxx  where TableName=xxx ' to query the task which executing the schema change DDL is inaccurate，  found that operation which  executing the schema change has a 'OK' result,but the query command has no result.

In summary, Whether the table structure change can be adjusted to synchronous execution or refactoring in a more elegant way?

**Comment by chaoyli** [comment]
If the schema change is synchronous, it will be cost much time to wait.

**Comment by chaoyli** [comment]
* Using the command 'SHOW ALTER TABLE COLUMN from xxx where TableName=xxx ' to query the task which executing the schema change DDL is inaccurate，i found that operation which executing the schema change has a 'OK' result,but the query command has no result. 
Could you reproduce it? If the result is OK but the query has no result, this a bug.
